UKHomeOffice/sgmr-service
# SPCR-110 Add descriptive error summary to People and Vessel tabs

## Description
Add descriptive error messages to the People and Vessel tab forms. Anchor the error messages to the div surrounding each input.

### To test
- Click on the people tab
- Click on 'save a person'
- Scroll to the bottom and click 'Add to saved people list'
- You should be taken to the error summary at top of page
- Click on the error messages
- They should take you to the top of the relevant inputs

- Repeats steps for the Vessel tab

## Developer Checklist

\* Required






Ensure you delete the branch once the Pull Request is merged.
